src/components/JobWizard/JobWizardSteps/JobWizardAdvanced/JobWizardAdvanced.js

```diff
@@ -65,19 +67,19 @@ const JobWizardAdvanced = ({ formState, stepIsActive }) => {
       )}
       <div className="form-row">
         <div className="form-col-1">
-          <FormInput name={`${ADVANCED_STEP}.inputPath`} label="Default input path" />
+          <FormInput id={`${ADVANCED_STEP}-input-path`} name={`${ADVANCED_STEP}.inputPath`} label="Default input path" />
```

Comment:
Probably better to have the `id` the same as `name` prop. Also I believe you can omit passing the `id`. We can use `name` as `id` inside the component. It's always unique